assets/src/js/cookies.js

```diff
@@ -16,36 +16,30 @@ export const setupCookies = () => {
     dataLayer.push(arguments);
   }
 
-  const updateGoogleConsent = (key, granted) => {
+  const updateGoogleConsent = (capabilities) => {
     if (!ENABLE_GOOGLE_CONSENT_MODE) {
```

Comment:
Not related to this implementation but maybe we can remove this statement here and check before invoking this method like we do [here](https://github.com/greenpeace/planet4-master-theme/pull/1621/files#diff-b2de966f19bbc1fc89c35c2de1f5ccae5590a86d6d4878a1a6f791a6350b410cR77)